README.md

```diff
@@ -10,7 +10,7 @@
 sccache - Shared Compilation Cache
 ==================================
 
-sccache is a [ccache](https://ccache.dev/)-like compiler caching tool. It is used as a compiler wrapper and avoids compilation when possible, storing cached results either on [local disk](#local) or in one of [several cloud storage backends](#storage-options).
+sccache is a [ccache](https://ccache.dev/)-like compiler caching tool. It is used as a compiler wrapper and avoids compilation when possible, storing cached results either on [local disk](docs/Local.md) or in one of [several cloud storage backends](#storage-options).
```

Comment:
What's the reason for chaning this? SHouldn't `#local` work just as well?

Reply:
The section "Local" had already been removed (prior to my PR). You can test that the link is dead by visiting either the local https://github.com/mozilla/sccache#local or in my repo https://github.com/exoosh/sccache#local ...